3ds Max/Max2Babylon/Forms/ExporterForm.cs

```diff
@@ -1,32 +1,32 @@
-﻿using Autodesk.Max;
-using BabylonExport.Entities;
-using System;
-using System.Diagnostics;
-using System.IO;
-using System.Threading.Tasks;
-using System.Windows.Forms;
-using Utilities;
-using Color = System.Drawing.Color;
-using Microsoft.WindowsAPICodePack.Dialogs;
-using System.Collections.Generic;
-using System.Linq;
-
-namespace Max2Babylon
-{
-    public partial class ExporterForm : Form
-    {
-        private const string ModelFilePathProperty = "modelFilePathProperty";
-        private const string TextureFolderPathProperty = "textureFolderPathProperty";
-
-        private readonly BabylonExportActionItem babylonExportAction;
-        private BabylonExporter exporter;
-        private bool gltfPipelineInstalled = true;  // true if the gltf-pipeline is installed and runnable.
-
-        TreeNode currentNode;
-        int currentRank;
-
-        private ExportItem singleExportItem;
-
+﻿using Autodesk.Max;
+using BabylonExport.Entities;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using Utilities;
+using Color = System.Drawing.Color;
+using Microsoft.WindowsAPICodePack.Dialogs;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Max2Babylon

```

Comment:
Looks like we have the same whitespace problem here as well, @noalak can your team please fix this as well? This should be good for checkin after :)

Reply:
Done